Fix connection method + leechers / seeders values

Naufrage7 opened the pull request:
Fixing connection method + seeders / leechers values ( wrong array index )
@adaur please check, i've tested and it's works for me

adaur commented:
It works but we should not be patching a bug coming from their side...
![ygg](https://user-images.githubusercontent.com/959078/30282297-daf002ca-96e2-11e7-83c4-8c554cd4d223.png)

Naufrage7 commented:
### Why i changed seeders / leechers array index ?
![capture du 2017-09-11 18-20-21](https://user-images.githubusercontent.com/11135217/30285554-c60e4dda-971e-11e7-8231-0e0b67fd2956.png)
**You used 3,4 as indexes but ( update maybe ? ) is 4 and 5 in fact. **
Please note that the site is not bugged ( from my side ), I am on yggtorrent.com but the language used is fr ( but no importance, is sickrage sending fr language code for web crawling ? )


### Why i changed login code ?
![capture du 2017-09-11 18-19-50](https://user-images.githubusercontent.com/11135217/30285773-7d898f92-971f-11e7-9d5f-4845c2b80f4f.png)
*Flow for valid credentials*

Useful lines are blue lines. The first request is for the login page (with images, form, ...), the second request is for posting data. When you post correct credentials, this page returns a refresh header like this: *refresh: 0; url = https: //yggtorrent.com/* so the browser redirects to home page and this corresponds to the third blue line.

**So, yes, the return of the form is a blank page.** When you post wrong credentials, the return page is the same as the login page ( check, in the first picture, the size is 103B - so, empty page - and on the second, the size is 13,2Kb - the login page with images, form ... - )


![capture du 2017-09-11 18-38-58](https://user-images.githubusercontent.com/11135217/30286166-e662cb0e-9720-11e7-8219-7568e7093e33.png)
*Flow for wrong credentials*

adaur commented:
OK, thanks for the clarification. It changed since I first wrote the code, and the blank page with the refresh tag is so fucked up... Strange I can still see the issue. Thanks anyway.
